lib/main.dart

```diff
@@ -114,47 +74,90 @@ class _MyHomePageState extends State<MyHomePage> {
   ];
 
   Future<void> _showChoiceDialog(BuildContext context) {
-    return showDialog(context: context, builder: (BuildContext context) {
-      return AlertDialog(
-        title: const Center(child: Text('Make a Choice...')),
-        content: SingleChildScrollView(
-          child: Row(
-            mainAxisAlignment: MainAxisAlignment.spaceAround,
-            children: <Widget>[
-              ElevatedButton(
-                  onPressed: () => pickImageFromGallery(),
+    return showDialog(
+      context: context,
+      builder: (BuildContext context) {
+        return AlertDialog(
+          title: const Center(
+            child: Text('Make a Choice...'),
+          ),
+          content: SingleChildScrollView(
+            child: Row(
+              mainAxisAlignment: MainAxisAlignment.spaceAround,
+              children: <Widget>[
+                ElevatedButton(
+                  onPressed: () {
+                    //pickImageFromGallery(),
+                  },
                   child: Icon(Icons.photo_album_outlined),
-                  style: ButtonStyle(
-                    shape: MaterialStateProperty.all(CircleBorder()),
-                    padding: MaterialStateProperty.all(EdgeInsets.all(10)),
-                    backgroundColor: MaterialStateProperty.all(Colors.cyan),
-                    overlayColor: MaterialStateProperty.resolveWith<Color?>((states) {
-                      if (states.contains(MaterialState.pressed)) return Colors.red;
-                    }),
+                  style: ElevatedButton.styleFrom(
+                    shape: CircleBorder(),
+                    padding: EdgeInsets.all(10),
+                    primary: Colors.cyan,
                   ),
-              ),
-              ElevatedButton(
-                  onPressed: () => pickImageFromGallery(),
+                ),
+                ElevatedButton(
+                  // change to better version in top
+                  onPressed: () {
+                    //pickImageFromGallery(),
+                  },
                   child: Icon(Icons.camera_alt_outlined),
                   style: ButtonStyle(
                     shape: MaterialStateProperty.all(CircleBorder()),
                     padding: MaterialStateProperty.all(EdgeInsets.all(10)),
                     backgroundColor: MaterialStateProperty.all(Colors.cyan),
-                    overlayColor: MaterialStateProperty.resolveWith<Color?>((states) {
-                      if (states.contains(MaterialState.pressed)) return Colors.red;
-                    }),
+                    overlayColor: MaterialStateProperty.resolveWith<Color?>(
+                      (states) {
+                        if (states.contains(MaterialState.pressed))
```

Comment:
use figure brackets in if statements. Only when on if condition  you wanna exit function without any result you can write is down as `if (condition) return;`

Reply:
fixed successfully